concepticondata/conceptlists/Carling-2017-240c.tsv

```diff
@@ -1,241 +1,240 @@
+Carling-2017-240c-107	107	1296	WALKING STICK	walking stick	12187	Artefacts	
+Carling-2017-240c-108	108	1462	POT	pot	12188	Artefacts	
+Carling-2017-240c-109	109	1358	ENEMY	enemy	12156	Social roles and positions	
+Carling-2017-240c-110	110	1072	PRISONER	captive, prisoner	12157	Social roles and positions	
+Carling-2017-240c-111	111	930	VILLAGE	village	12252	Social roles and positions	
+Carling-2017-240c-112	112	789	PEOPLE	people	12253	Social roles and positions	
+Carling-2017-240c-113	113	302	CLAN	tribe, clan	12254	Social roles and positions	
+Carling-2017-240c-114	114	301	CHIEFTAIN	tribal chief	12255	Social roles and positions	
+Carling-2017-240c-115	115	1325	FRIEND	friend	12256	Social roles and positions	
+Carling-2017-240c-116	116	382	GOVERN	to govern	12257	Social roles and positions	
+Carling-2017-240c-117	117	1508	KING	king, governor	12258	Social roles and positions	
+Carling-2017-240c-118	118	1306	QUEEN	queen	12259	Social roles and positions	
+Carling-2017-240c-119	119			lord, boss, proprietor	12260	Social roles and positions	
+Carling-2017-240c-120	120	812	SLAVE	slave	12261	Social roles and positions	
+Carling-2017-240c-121	121	255	SERVANT	servant	12262	Social roles and positions	
+Carling-2017-240c-122	122	384	FREEMAN	free man	12263	Social roles and positions	
+Carling-2017-240c-123	123	1128	ORDER	to order, to command	12264	Social roles and positions	
+Carling-2017-240c-124	124	1128	ORDER	to give order	12265	Social roles and positions	
+Carling-2017-240c-125	125	1635	OBEY	to obey	12266	Social roles and positions	
+Carling-2017-240c-126	126	1369	GOLD	gold	12205	Metals	
+Carling-2017-240c-127	127	759	SILVER	silver	12206	Metals	
+Carling-2017-240c-128	128	386	BATTLE-AXE	battle-axe	12160	Warfare	
+Carling-2017-240c-129	129	579	BATTLE	battle	12161	Warfare	
+Carling-2017-240c-130	130	994	BOW	bow	12162	Warfare	
+Carling-2017-240c-131	131	977	ARROW	arrow	12163	Warfare	
+Carling-2017-240c-132	132	945	SPEAR	spear	12164	Warfare	
+Carling-2017-240c-133	133	1172	SHOOT	to shoot	12165	Warfare	
+Carling-2017-240c-134	134	1763	CLUB	club	12166	Warfare	
+Carling-2017-240c-135	135	1417	KILL	to kill	12326	Warfare	
+Carling-2017-240c-136	136	1358	ENEMY	enemy	12327	Warfare	
+Carling-2017-240c-137	137	1423	FIGHT	to fight	12328	Warfare	
+Carling-2017-240c-138	138	935	WAR	war	12329	Warfare	
+Carling-2017-240c-139	139	1338	PEACE	peace	12330	Warfare	
+Carling-2017-240c-140	140	1561	ARMY	army	12331	Warfare	
+Carling-2017-240c-141	141	783	VICTORY	victory	12332	Warfare	
+Carling-2017-240c-142	142	782	DEFEAT	defeat (n)	12333	Warfare	
+Carling-2017-240c-143	143	722	ATTACK	to attack	12334	Warfare	
+Carling-2017-240c-144	144	1330	DEFEND	to defend	12335	Warfare	
+Carling-2017-240c-145	145	1072	PRISONER	captive, prisoner	12336	Warfare	
+Carling-2017-240c-146	146	65	AMBUSH	ambush (n)	12337	Warfare	
+Carling-2017-240c-147	147			trumpeter bird (watchbird)	12338	Warfare	
+Carling-2017-240c-148	148		*ARROW TIP	arrow tip	12339	Warfare	
```

Comment:
This was just discussed in [#443](https://github.com/clld/concepticon-data/issues/443), where we discussed to add this concept. I follow @Schweikhard's idea mentioned there. So you could already use what he suggested as wording.